frontend/public/components/utils/documentation.tsx

```diff
@@ -1,3 +1,9 @@
 // Prefer the documentation base URL passed as a flag, but fall back to the latest docs if none was specified.
 export const openshiftHelpBase =
   window.SERVER_FLAGS.documentationBaseURL || 'https://docs.okd.io/latest/';
+
+export const networkPolicyDocs = (openshiftFlag: boolean) => {
+  return openshiftFlag
+    ? `${openshiftHelpBase}networking/configuring-networkpolicy.html'`
```

Comment:
You have an extra `'`

```suggestion
    ? `${openshiftHelpBase}networking/configuring-networkpolicy.html`
```

Reply:
How do you spot stuff like that? :)